multimodal/tests/unittests/others/test_classification.py

```diff
@@ -57,3 +57,33 @@ def test_focal_loss_multiclass(checkpoint_name):
     predictions_str = predictor.predict(image_path)
     predictions_list1 = predictor.predict([image_path])
     predictions_list10 = predictor.predict([image_path] * 10)
+
+
+@pytest.mark.parametrize(
+    "checkpoint_name,eval_metric",
+    [
+        ("swin_tiny_patch4_window7_224", "log_loss"),
+        ("swin_tiny_patch4_window7_224", "f1_micro"),
+    ],
+)
+def test_metrics_multiclass(checkpoint_name, eval_metric):
+    download_dir = "./ag_automm_tutorial_imgcls"
+    train_data, test_data = shopee_dataset(download_dir)
+
+    model_path = f"./tmp/{uuid.uuid4().hex}-automm_shopee"
+
+    predictor = MultiModalPredictor(label="label", problem_type="multiclass", eval_metric=eval_metric, path=model_path)
+
+    predictor.fit(
+        hyperparameters={
+            "model.mmdet_image.checkpoint_name": checkpoint_name,
+            "env.num_gpus": -1,
+            "optimization.max_epochs": 1,
+        },
+        train_data=train_data,
+        time_limit=30,  # seconds
+    )  # you can trust the default config, e.g., we use a `swin_base_patch4_window7_224` model
+
+    score = predictor.evaluate(train_data)
```

Comment:
should you assert-check the score?

Reply:
It's not deterministic, especially when we have a time_limit of 30 seconds.

Reply:
Since this issue is relevant to the correctness of metric calculation, it would be better if there is some sort of guardrail here.

Reply:
Perhaps get the train predictions via `predictor.predict / predict_proba`, then ensure manual metric calculation matches `evaluate` output?

Reply:
The above is what I do in tabular's tests

Reply:
Added an assert-check to the scores